web/src/bootstrap_typeahead.ts

```diff
@@ -467,10 +491,11 @@ export class Typeahead<ItemType extends string | object> {
     }
 
     unlisten(): void {
+        this.hide();
         this.$container.remove();
-        const events = ["blur", "keydown", "keyup", "keypress", "mousemove"];
```

Comment:
Swapping this to `click` does seem like a bug fix; `listen` adds a mousemove handler but to the menu element, which is presumably removed when removing `this.$container`?

Reply:
Yeah